app/2.0.x/cli.md

```diff
@@ -1,4 +1,9 @@
 ---
+#
+#  WARNING: this file was auto-generated by a script.\n")
```

Comment:
I notice the line break characters (I think?) (`\n")` ) are displaying in some of these warnings, though not all. Any way you could remove them without having problems?

Reply:
Oh, my. I am so clumsy. Thanks for reporting this, I will remove those.

Reply:
This should be fixed now.